packages/stylelint-plugin/test.scss

```diff
@@ -1,22 +1,17 @@
-@import "~@kaizen/design-tokens/sass/spacing";
 // This is a file you can use to play with the stylelint plugin
+@import "~@kaizen/design-tokens/sass/spacing";
 @import "~@kaizen/design-tokens/sass/spacing-vars";
-
 @import "~@kaizen/design-tokens/sass/color-vars";
-@import "././another.scss";
-
 @import "~@kaizen/component-library/styles/layout";
 
-$test-color: $kz-var-color-wisteria-800;
+$test-color: $kz-var-color-purple-800;
 
-/* $test-offset: -1 * ($kz-spacing-md + 1px);
- */
 .test {
-  background-color: rgb($kz-var-color-cluny-600-rgb-params);
+  background-color: rgb($kz-var-color-blue-600-rgb-params);
```

Comment:
This whole file need a cleanup as they'll be stuck in an unfixable state (neither the old token names or the new ones)

Reply:
@jasononeil i think i might just delete these test files...